### PR TITLE
39). Add topic files 123 to 126

### PR DESCRIPTION
**Pull request by jameshkramer**
This PR is for user story #1576037.

It adds four article files, makes corresponding deletions from the C# and VB folders, and adds redirection entries. It also provides fixes to toc.yml and some article files.

Please merge #18413, #18439, #18541, #18580, #18589, and #18673 before merging this.

cc: @BillWagner @mairaw @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

**Comment by Thraka** [comment]
Just curious, why do we care if there is or isn't quotes around the title?

**Comment by jameshkramer** [comment]
@Thraka, my reason for removing the quotes is not very substantial, but here it is. Quotes are only required around the title (or description) if there is a colon in it. So, if they are not required, I prefer they not be there, consistently. What it comes down to is simply that I like consistency in general, and in this case just because it is neater.

**Comment by BillWagner** [comment]
@jameshkramer After a quick look, this is fine.  I'll review more carefully after resolving the conflicts.

**Comment by mairaw** [comment]
@jameshkramer can you resolve the conflicts here so we can review and merge? Thx!

**Comment by jameshkramer** [comment]
@mairaw 

The conflicts are resolved

File: docs/standard/linq/atomized-xname-xnamespace-objects.md [hunk 1]
```diff
@@ -39,9 +60,9 @@ Else
 End If
 ```
 
-This example produces the following output:
+ This example produces the following output:
```
Comment (mairaw) [inline docs/standard/linq/atomized-xname-xnamespace-objects.md:63]:
```suggestion
This example produces the following output:
```

File: docs/standard/linq/wordprocessingml-document-styles.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ ms.assetid: 40e35de6-ac93-4bba-88ab-a018cbe93873
...
 
-To understand styles it is helpful to know something about document structure. A WordprocessingML document is stored in a *package* that comprises *parts*. Specifically, the term *package* corresponds to a ZIP archive and the term *part* corresponds to a file stored within the ZIP. If a document contains paragraphs that are formatted with styles, there will be a document part that contains paragraphs that have styles applied to them, and a style part that defines the styles referenced in the document part.
+To understand styles it's helpful to know something about document structure. A WordprocessingML document is stored in a *package* that comprises *parts*. Specifically, the term *package* corresponds to a ZIP archive and the term *part* corresponds to a file stored within the ZIP. If a document contains paragraphs that are formatted with styles, there will be a document part that contains paragraphs that have styles applied to them, and a style part that defines the styles referenced in the document part.
```
Comment (mairaw) [inline docs/standard/linq/wordprocessingml-document-styles.md:15]:
```suggestion
To understand styles, it's helpful to know something about document structure. A WordprocessingML document is stored in a *package* that comprises *parts*. Specifically, the term *package* corresponds to a ZIP archive and the term *part* corresponds to a file stored within the ZIP. If a document contains paragraphs that are formatted with styles, there will be a document part that contains paragraphs that have styles applied to them, and a style part that defines the styles referenced in the document part.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Some quick suggestions to address

**Review by mairaw — APPROVED** [review]
LGTM

**Pull request merged**